cbor/src/test/java/com/fasterxml/jackson/dataformat/cbor/fuzz/Fuzz35979ShortTextTest.java

```diff
@@ -0,0 +1,32 @@
+package com.fasterxml.jackson.dataformat.cbor.fuzz;
```

Comment:
There is already 

com/fasterxml/jackson/dataformat/cbor/failing/Fuzz_35979_StringValueTest.java